pkg/sdkserver/localsdk.go

```diff
@@ -89,9 +91,10 @@ type LocalSDKServer struct {
 
 // NewLocalSDKServer returns the default LocalSDKServer
 func NewLocalSDKServer(filePath string) (*LocalSDKServer, error) {
+	gs := defaultGs()
 	l := &LocalSDKServer{
 		gsMutex:         sync.RWMutex{},
-		gs:              defaultGs,
+		gs:              gs,
```

Comment:
```suggestion
		gs:              defaultGs(),
```

Having it inline seems nicer?